### ISSUE TITLE
Add metadata for CINECA

### ISSUE THREAD

**Issue by jamesaoverton** [jamesaoverton/IHCC#3]
We should include various metadata in each of the OWL files we build. For starters, how about a `metadata/cineca.ttl` file like this, that we can merge into the results:

```
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix dcterms: <http://purl.org/dc/terms/>.
@prefix prov: <http://www.w3.org/ns/prov#> .

<>
  rdf:type owl:Ontology ;
  dcterms:title "TODO: A short title" ;
  dcterms:description "TODO: A human-readable description" ;
  dcterms:license "TODO: The license URL" ;
  dcterms:rights "TODO: A human-readable description of rights" ;
  prov:wasDerivedFrom <https://docs.google.com/spreadsheets/d/1ZXqTMIhFtGOaodw7Fns5YghvY_pWos-RuSa2BFnO5l4> .
```

The `Makefile` should also add `owl:versionIRI` and `dcterms:date`.

We could get fancier with [PROV](https://www.w3.org/TR/prov-primer/) modelling.

@mcourtot Can you or your team please help fill some of these fields?

**Comment by mcourtot** [comment]
fantastic - love the ttl.

is dcterms:rights just a more human readable version than the license URL?

**Comment by jamesaoverton** [comment]
That's my understanding based on <https://www.dublincore.org/specifications/dublin-core/dcmi-terms/> which says: (license) "Recommended practice is to identify the license document with a URI. If this is not possible or feasible, a literal value that identifies the license may be provided" and (rights) "Typically, rights information includes a statement about various property rights associated with the resource, including intellectual property rights. Recommended practice is to refer to a rights statement with a URI. If this is not possible or feasible, a literal value (name, label, or short text) may be provided".

But there are multiple versions of DC and multiple standards (PROV, CC), and honestly I find it all confusing.

OBO current practise is a `dcterms:license` URL and `rdfs:comment` for rights. But last week on the OBO issue tracker someone recommended `cc:useGuidelines`: 

- http://obofoundry.org/principles/fp-001-open.html
- https://creativecommons.org/ns#useGuidelines
- https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1019#issuecomment-591345269

I was trying to save you some work my making these little decisions, but I'm more than happy to change it to whatever you want! 😄

**Comment by mcourtot** [comment]
saving work sounds great ;)
I had not seen dcterms:rights before, thus the question. We'll go for CC-BY, happy with whatever your preferred encoding.

side note not relevant to IHCC: wondering if we need both - what happens if there is contradiction? have added to the OBO tracker...

**Comment by jamesaoverton** [comment]
@mcourtot Do you have preferences for the other TODO items: title, description?

**Comment by mcourtot** [comment]
Title: Genomics Cohorts Knowledge Ontology (GECKO)

Description: An ontology to represent genomics cohort attributes. The GECKO is maintained by the CINECA project, https://www.cineca-project.eu, and standardises attributes commonly used for genomics cohort description as well as individual-level data items. A series of tools is being developed to enable automated generation of harmonised data files based on a JSON schema mapping file. For more information please contact info@cineca-project.eu

License: CC-BY

**Comment by jamesaoverton** [comment]
@beckyjackson Please implement this, and rename everything except `data/cineca.tsv` from "cineca" to "gecko".

@mcourtot This document about ontology metadata showed up on obo-discuss. If you want richer metadata, let's use it as a starting point: https://www.isibang.ac.in/ns/mod/index.html

**Comment by jamesaoverton** [comment]
We'll count this as closed by #12, and revise as needed.
